common/system/filtering/filters.cpp

```diff
@@ -89,12 +89,22 @@ static double filterAccelOneAxis(double* accelReadings, double* imuReadings,
                                  const SensorProperties_s* sensorProperties) {
   int numAccelsValid = 0;
   double accelSum = 0;
+  int highestFsIndex = 0;
 #if HAS_DEV(ACCEL) || HAS_DEV(IMU)
   int highestValidPriority = 0;
 #endif  // HAS_DEV(ACCEL) || HAS_DEV(IMU)
   // Only pull data if accel below fullscale and sensor is working
 #if HAS_DEV(ACCEL)
   for (int i = 0; i < NUM_ACCEL; i++) {
+    if (sensorProperties->accelFs[i] >
+        sensorProperties->accelFs[highestFsIndex]) {
+      highestFsIndex = i;
```

Comment:
Right now the `highestFsIndex` is used as an index into both the accelerometer and imu arrays which might turn into an out of bounds error.